### PR TITLE
Add endpoint to flush metrics cache

### PR DESCRIPTION
**Pull request by michielderoos**
**Describe the Pull Request**

Sometimes if we change the past, the cache gets invalidated and we get some strange behaviour. This adds an endpoint to nuke the cache for a given org if we have to

**Todo**

- [x] Link relevant [trello card](https://trello.com/b/PA2LhlOh/sempo-dev) or [related issue](https://github.com/teamsempo/SempoBlockchain/issues) to the pull request
- [x] Request review from relevant @person or team
- [x] QA your pull request. This includes running the app, login, testing changes, checking [accessibility](https://www.a11yproject.com/checklist/) etc.
- [x] Bump [backend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/config.py#L7) and/or [frontend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/app/package.json#L3) versions following Semver

### CODE REVIEW

**Comment by michielderoos** [comment]
[Add metrics cache flush](https://trello.com/c/WWITQ7ED/731-add-metrics-cache-flush)

File: app/server/api/metrics_api.py [hunk 1]
```diff
@@ -130,6 +130,24 @@ def get(self):
 
         return make_response(jsonify(response_object)), 200
 
+class CacheApi(MethodView):
+    @requires_auth(allowed_roles={'ADMIN': 'any'})
```
Comment (tristanhcole) [inline app/server/api/metrics_api.py:134]: feels like this should be restricted to sempoadmin

Comment (michielderoos) [reply app/server/api/metrics_api.py:134]: Done

**Review by tristanhcole — APPROVED** [review]
Tested and works for me. Would still be good to have a separate PR to resolve the issue of weekly withdrawals going from PENDING to COMPLETE a few days later, but the cache not resetting.

**Pull request merged**